### PR TITLE
Complete The HW 4

### PR DESCRIPTION
**Pull request by AnastasiiaNashchokina**

### CODE REVIEW

File: src/main/java/AN_hw_4/Task1.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter the coefficients a, b, and c");
+        System.out.println("a: ");
+        double a = scanner.nextDouble();
+        System.out.print("b: ");
+        double b = scanner.nextDouble();
+        System.out.print("c: ");
+        double c = scanner.nextDouble();
+        double D = b * b - 4 * a * c;
```
Comment (AntBelyi) [inline src/main/java/AN_hw_4/Task1.java]: bad name for 'D' variable. First of all - all variables should begin from the small letter.
Lets rename it as discriminator

File: src/main/java/AN_hw_4/Task2.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                min = numbers[i];
+            }
+            if (numbers[i] > max) {
+                max = numbers[i];
+            }
+        }
+            System.out.println("Minimum value: " + min);
+            System.out.println("Maximum value: " + max);
+    }
+}
```
Comment (AntBelyi) [inline src/main/java/AN_hw_4/Task2.java:27]: Tabulation should be fixed.
Screenshot_14

File: src/main/java/AN_hw_4/Task2.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        Scanner scanner = new Scanner(System.in);
+            System.out.println("Enter a series of integers separated by spaces:");
+        String input = scanner.nextLine();
+        String[] number = input.split(" ");
+        int[] numbers = new int[number.length];
+        for (int i = 0; i < number.length; i++) {
+            numbers[i] = Integer.parseInt(number[i]);
+        }
+        int min = numbers[0];
+        int max = numbers[0];
```
Comment (AntBelyi) [inline src/main/java/AN_hw_4/Task2.java]: Lets rename as minValue and maxValue

File: src/main/java/AN_hw_4/Task3.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        System.out.println(text);
+        text = text.toLowerCase();
+        String[] words = text.split(" ");
+        int count = 0;
+        for (String word : words) {
+            if (word.equals("the")) {
+                count++;
+            }
+        }
+        System.out.println("The word 'the' occurs " + count + " times in the text.");
```
Comment (AntBelyi) [inline src/main/java/AN_hw_4/Task3.java]: I can see four words 'the'
Screenshot_15

But your output is:
Screenshot_16

Try to implement a Scanner. Whenever substring i type into - your program should count how much entered from console substrings in your text

File: src/main/java/AN_hw_4/Task1.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (AntBelyi) [inline src/main/java/AN_hw_4/Task1.java]: Please remove all empty lines
Screenshot_17

File: src/main/java/AN_hw_4/Task3.java (hunk 4 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/AN_hw_4/Task3.java]: fixed

File: src/main/java/AN_hw_4/Task2.java (hunk 2 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/AN_hw_4/Task2.java:27]: fixed

File: src/main/java/AN_hw_4/Task2.java (hunk 3 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/AN_hw_4/Task2.java]: fixed

File: src/main/java/AN_hw_4/Task1.java (hunk 1 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/AN_hw_4/Task1.java]: fixed

File: src/main/java/AN_hw_4/Task1.java (hunk 5 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/AN_hw_4/Task1.java]: fixed

**Pull request merged**